src/main/java/org/cdc/gov/sdp/queue/DatabaseQueueProducer.java

```diff
@@ -53,35 +64,45 @@ public void process(final Exchange exchange) throws Exception {
 				queueConnection = queueDataSource.getConnection();
 			}
 
+			if (!tableCreated) {
```

Comment:
Is there a way to do this in the component\endpoint so it can be done for both the producer and consumer without having to check each time we go to access the table?